test/cases/insert-function/insert-function.test.js

```diff
@@ -0,0 +1,33 @@
+/* globals document, getComputedStyle */
+
+import path from 'path';
+
+import webpack from 'webpack';
+
+import config from './webpack.config';
+
+describe('options.insert as a function', () => {
+  it('inserts the bundle on the page', (done) => {
+    const outputPath = path.resolve(__dirname, 'expected/index.js');
+    webpack({
+      ...config,
+      output: {
+        path: outputPath,
+        libraryTarget: 'umd',
+        library: 'mini',
+      },
+    }).run(() => {
+      let computedValue = getComputedStyle(document.body).backgroundColor;
+      expect(computedValue).toBe('');
+
+      const script = document.createElement('script');
+      script.src = outputPath;
```

Comment:
@evilebottnawi it seems that `webpack` is not emitting this file and it 404s. Also I am not sure jsdom supports `getComputedStyle`. In that case I think we either get rid of this test or need to set up something like Puppeteer which IMO is out of scope.